pyglet/graphics/vertexdomain.py

```diff
@@ -292,9 +306,29 @@ def migrate(self, domain):
         self.domain.allocator.dealloc(self.start, self.count)
         self.domain = domain
         self.start = new_start
+        self._cache_version = None
+
+    def get_setter(self, name):
+        """
+        Returns a function that, when called, sets the values for a given attribute.
+
+        The setter becomes invalid when the VertexList migrates to a different
+        domain, and it's the caller's responsibility to stop using the old setter
+        and create a new one.
+        """
```

Comment:
1. Same docstring style nitpick as before
2. Phrasing tweaks
3. We can make debugging easier by explaining what happens when you fail to migrate to the new setter so the issue can be identified
```suggestion
        """Return a function which sets the value(s) of the named attribute.

        The setter is invalidated whenever the VertexList migrates to a different
        domain. It's the caller's responsibility to stop using the old setter and
        create a new one when this happens.
        """
```